man/fi_endpoint.3.md

```diff
@@ -1431,12 +1431,15 @@ be associated with a single shared transmit context.
 Unlike shared transmit contexts, applications interact directly with
 shared receive contexts.  Users post receive buffers directly to a
 shared receive context, with the buffers usable by any endpoint bound
-to the shared receive context.  Shared receive contexts are not
-associated with completion queues or counters.  Completed receive
-operations are posted to the CQs bound to the endpoint.  An endpoint
-may only be associated with a single receive context, and all
-connectionless endpoints associated with a shared receive context must
-also share the same address vector.
+to the shared receive context.  Applications must allocate and bind
+shared transmit contexts to endpoints, but operations are posted
+directly to the endpoint. An SRX must be bound to a receive CQ in order
```

Comment:
We can't make this a *must* without breaking ABI compatibility. I would reword with *may*.  This change requires updates to the providers.